lib/SILOptimizer/IPO/CrossModuleSerializationSetup.cpp

```diff
@@ -181,12 +182,17 @@ makeSubstUsableFromInline(const SubstitutionMap &substs) {
     }
   }
 }
+static llvm::cl::opt<bool> SerializeEverything("sil-cross-module-serialize-all",
+                                               llvm::cl::default(false),
```

Comment:
I don't know what I'm doing here, but the Windows compiler very sensibly rejects this use of the default keyword.
```suggestion
                                               llvm::cl::init(false),
```
@gottesmm, WDYT?